projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -80,6 +81,8 @@ const VideoPressEdit = CoreVideoEdit =>
 			const { guid } = this.props.attributes;
 			if ( ! guid ) {
 				await this.setGuid();
```

Comment:
is there a chancethat this call will throw? Should we wrap in a try/catch?

Reply:
From looking at `setGuid`, I think it doesn't throw, but it does `fall back` to the core block if something goes wrong. So I think safe to leave as-is.